build.rs

```diff
@@ -1,30 +1,31 @@
 // https://github.com/longbridgeapp/rust-i18n/blob/v0.1.6/crates/support/src/lib.rs#L9
 fn workdir() -> Option<String> {
-    // Do not panic if OUT_DIR or workdir not correct.
-    // Because in cargo install OUT_DIR is:
-    // /var/folders/qb/.../T/cargo-installcBf9L5/release/build/rust-i18n-xxx/build-script-build
-    let dest = std::env::var("OUT_DIR");
-    if dest.is_err() {
-        return None;
-    }
-    let dest = dest.unwrap();
+    let workdir = std::env::var("PWD");
```

Comment:
@huacnlee I think this could be a more robust solution, the PWD environment variable will always be found and it will work building locally as inside a docker container and it even worked for me in a github action, I would appreciate if you can test it and tell me if this is correct.